src/lang/en-US.ts

```diff
@@ -663,6 +670,34 @@ const Supabase: ResolvedIntlConfig['messages'] = {
     'supabase.poller.failed': `We encountered a problem retrieving the status of this action. Please check your network connection and try again.`,
 };
 
+const Legal: ResolvedIntlConfig['messages'] = {
+    'legal.heading': `Legal Stuff`,
+    'legal.heading.outdated': `Updated Legal Stuff`,
+    'legal.message': `Please use the links below to open and review the documents before checking accept and continuing.`,
```

Comment:
For both legal.message and legal.message. outdated, something about "...before checking accept and continuing" feels a bit awkward. Can it just be "...before you continue."?  It's pretty clear that they have to check the box and they get the error when they don't, anyway.